android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -138,11 +140,12 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
     @ReactMethod
     fun isFingerprintAuthenticatorRegistered(profileId: String?, promise: Promise) {
         profileId ?: promise.rejectWithNullError("profileId", "String").run { return }
-        try {
-            promise.resolve(authenticatorManager.isFingerprintAuthenticatorRegistered(profileId))
-        } catch (e: OneginiError) {
-            promise.reject(e.errorType.toString(), e.message)
+        authenticatorManager.getUserProfile(profileId)?.let { userProfile ->
```

Comment:
This is the only file with those changes, but it's just in alot of locations and the file is really big. This will cause annoying conflicts that we can prevent. 

The ticket has been created, yes.